test/behavior/tuple.zig

```diff
@@ -338,7 +338,7 @@ test "zero sized struct in tuple handled correctly" {
                     .type = struct {},
                     .default_value_ptr = null,
                     .is_comptime = false,
-                    .alignment = 0,
+                    .alignment = @alignOf(struct {}),
```

Comment:
Bad idea; this type is distinct from the one declared a few lines up, so the compiler *could* give it a different alignment if it felt so inclined (though our current implementation never will). Extract this into a separate `const Inner = struct {};` or something.